.gitignore

```diff
@@ -12,3 +12,8 @@
 *.pyc
 *.egg-info
 .pytest_cache
+
+
+# virtual env
+venv_*
```

Comment:
Thanks a lot for the PR.

First, instead of venv_*, I think it would be better to exclude the whole `venv` folder.

Looking forward to the changes